files/system/usr/share/ublue-os/just/60-custom.just

```diff
@@ -78,6 +78,70 @@ harden-flatpak:
         flatpak override --user --env=LD_PRELOAD=/var/run/host/usr/lib64/glibc-hwcaps/x86-64-v"$bestuarch"/libhardened_malloc.so
     fi
 
+# Check for unverified flatpak repos and unsafe permissions
+audit-flatpak:
+	#!/bin/bash
+	function hasPermission {
+		local permissions=$1
+		local prefix=$2
+		local query=$3
+		local line=$(grep "^${prefix}=" <<< "$permissions" | sed -e "s/^${prefix}=//" -e "s/#.*//")
+		IFS=';' read -r -a list <<< "$line"
+		for p in ${list[@]}; do
+			if [[ "$p" == "$query" ]]; then
+				return
+			fi
+		done
+		return 1
+	}
+	
+	echo "Checking remotes..."
+	remotes="$(flatpak remotes -d)"
+	while read -r remote ; do
+		ref="$(cut -f 1 <<<"$remote")"
+		url="$(cut -f 3 <<< "$remote")"
+		subset="$(cut -f 5 <<< "$remote")"
+		if [ "$ref" != "flathub-verified" ]; then
+			echo "Warning: unverified remote found: $ref"
+		else 
+			if [ "$subset" != "verified" ]; then
+				echo "Warning: flathub-verified is improperly configured: subset is set incorrectly"
+				echo "$subset"
+			fi
+			if [ "$url" != "https://dl.flathub.org/repo/" ]; then
```

Comment:
one thing is that this doesn't support the verifiedbeta repo, which isn't terrible but right now it's the only way to use GIMP on wayland (via the gimp beta)

otherwise this looks good!